.gitignore

```diff
@@ -56,6 +56,7 @@ lcov*.info
 /*.pc
 /*.rc
 /*_prelude.c
+/build*
```

Comment:
should be separate commit.

Reply:
Why `/build*` is ignored? If this is because of your build configuration, it should be in `.git/info/exclude` or `~/.config/git/ignore`.

Reply:
I use it on 4 different machines, so it seemed like a very harmless thing to add that makes my life much easier.

Reply:
To make your life easier, you introduced a trap which accidentally ignores a file if someone added a new file whose name starts with "build".

Reply:
How about changing it to `/build*/` which only ignores directories?

Reply:
It's better but is `*` required?

Reply:
I often build multiple branches Into the same repository, e.g. `/build-master` `/build-branch1`, `/build-branch2` and so on. So, I do find it useful. What do you think? Is it okay?

Reply:
I still think it's bad manner but anyway `/build*/` is much better.